src/main/io/displayport_msp_bf_compat.c

```diff
@@ -38,7 +38,7 @@ uint8_t getBfCharacter(uint8_t ch, uint8_t page)
         return BF_SYM_RSSI;
 
     case SYM_LQ:
-        return BF_SYM_AH_LEFT;
```

Comment:
BF_SYM_QUALITY is the correct value here. It is a bug in DJI's font.

Reply:
Yeah, this one I chose a symbolic replacement from what was available. I do agree the correct symbol would be much better, but let's work with what we have right now.